### PR TITLE
Nacos support subscribe to all with '*'

### PR DESCRIPTION
**Pull request by FinalT**

### CODE REVIEW

File: registry/nacos/registry.go [hunk 1]
```diff
@@ -169,36 +169,95 @@ func (nr *nacosRegistry) Subscribe(url *common.URL, notifyListener registry.Noti
...
+				logger.Warnf("event listener game over.")
+				return perrors.New("nacosRegistry is not available.")
+			}
 
-	for {
-		if !nr.IsAvailable() {
-			logger.Warnf("event listener game over.")
-			return perrors.New("nacosRegistry is not available.")
-		}
+			services, err := nr.getAllSubscribeServiceNames()
```
Comment (chickenlj) [inline registry/nacos/registry.go:181]: Do we need a scheduled pulling task here? What if the service name list in Nacos server changes after we subscribed to '*', for example, if a new service name is registered, will we still get notified by subscribing to '*'?

**Review by binbin0325 — COMMENTED** [review]
In what scenarios should subscribe to all services

File: registry/nacos/registry.go (hunk 1 above)
Comment (chickenlj) [reply registry/nacos/registry.go:181]: This can be added later.

**Review by chickenlj — APPROVED** [review]
LGTM.

**Pull request merged**